plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt

```diff
@@ -24,9 +26,33 @@ fun androidLibraryFeatureDefinition(
     pluginName = "com.android.library",
     pluginExtension = LibraryExtension::class,
     featureConfiguration = featureConfiguration,
-    configuration = { extension, feature, _, formaConfiguration ->
+    configuration = { extension, feature, project, formaConfiguration ->
         with(extension) {
             compileSdkVersion(formaConfiguration.compileSdk)
+            if (feature.generateManifest) {
+                /**
+                 * Some naive caching here, file name equals package name
+                 * We create new file only if package is changed or on first build
+                 */
+                with(File(project.buildDir, "tmp/manifest/${feature.packageName}.xml")) {
+                    if (!exists()) {
+                        parentFile.mkdirs()
+                        createNewFile()
+                        writeText(
+                            """<?xml version="1.0" encoding="utf-8"?>
+                           <manifest package="${feature.packageName}"/>
```

Comment:
What about support theme for module? We have to add themes for correct layout/resources preview in android studio 
```xml
<application android:theme="@style/AppTheme.Mobile"/>
```

Reply:
You can choose the theme on the preview screen, no?

Reply:
<img width="680" alt="Screen Shot 2021-01-31 at 10 41 47 PM" src="https://user-images.githubusercontent.com/1287797/106423586-63953a80-6469-11eb-8438-4fb94ff6fc1a.png">